sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -14,6 +14,29 @@ include::modules/sandboxed-containers-preparing-openshift-cluster.adoc[leveloffs
 .Additional resources
 * xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user-provisioned cluster on bare metal]
 
+[id="checking-nodes-sandboxed-containers"]
+== Checking if you can deploy {sandboxed-containers-first} in your cluster
+
+Before running {sandboxed-containers-first} containers, you need to check whether the nodes in your cluster are eligible to run Kata containers. Some cluster nodes might not comply with sandboxed containers' minimum requirements. The most common reason for node ineligibility is the lack of virtualization support on the node. If you attempt to run sandboxed workloads on ineligible nodes, you will experience errors. You can use the Node Feature Discovery Operator (NFD) and a `NodeFeatureDiscovery` resource to automatically check node eligibility. When installed, NFD prevents installation of the sandboxed runtime on ineligible nodes.
```

Comment:
Remove "containers" after {sandboxed-containers-first} or this will render as "OpenShift sandboxed containers containers".

Reply:
I'm not sure whether checking eligibility is a requirement, or an option.  If they aren't required to check, then I would change "you need to check" to "you can check".

Reply:
@MirzWeiss It's an option (see also [here](https://github.com/openshift/openshift-docs/pull/42831#discussion_r822431859)). If a user knows their workers are eligible they are free to skip the check.  OTOH if they do skip the check and then it turns out they were wrong in their eligibility assessment they'll run into problems that might be hard to understand for some users.

Reply:
Updated